src/main/java/org/tikv/common/TiConfiguration.java

```diff
@@ -689,4 +695,36 @@ public int getKeepaliveTimeout() {
   public void setKeepaliveTimeout(int timeout) {
     this.keepaliveTimeout = timeout;
   }
+
+  public boolean isTlsEnable() {
+    return tlsEnable;
+  }
+
+  public void setTlsEnable(boolean tlsEnable) {
+    this.tlsEnable = tlsEnable;
+  }
+
+  public Optional<String> getTrustCertCollectionFile() {
+    return trustCertCollectionFile;
+  }
+
+  public void setTrustCertCollectionFile(Optional<String> trustCertCollectionFile) {
```

Comment:
Is it normal to use Optional<T> as a method argument? It seems like this is discouraged: https://rules.sonarsource.com/java/RSPEC-3553

Reply:
fixed, PTAL